cypress.json

```diff
@@ -1 +1,4 @@
-{}
+{
+  "baseUrl": "http://192.168.100.6:5601",
```

Comment:
this url shouldn't be here.
add this this .json file to gitignore and create a dummy file with examples

Reply:
Yes, i'm going to try to implement the dotenv module. Thank you